portfolio/src/main/webapp/script.js

```diff
@@ -216,10 +216,13 @@ async function createBigfootSightingsMap() {
     document.getElementById('map'),
     { center: { lat: 39.8283, lng: -98.5795 }, zoom: 4 });
 
+  let markers = [];
   bigfootSightings.forEach((bigfootSighting) => {
-    new google.maps.Marker(
-      { position: { lat: bigfootSighting.latitude, lng: bigfootSighting.longitude }, map: map });
+    markers.push(new google.maps.Marker(
+      { position: { lat: bigfootSighting.latitude, lng: bigfootSighting.longitude }, map: map }));
   });
+
+  new MarkerClusterer(map, markers, { imagePath: '/images/markers/' });
```

Comment:
nit: this looks strange, do we not need to return it or save this variable somewhere?